content/engine/release-notes/24.0.md

```diff
@@ -20,6 +20,36 @@ For more information about:
 - Deprecated and removed features, see [Deprecated Engine Features](../deprecated.md).
 - Changes to the Engine API, see [Engine API version history](../api/version-history.md).
 
+## 24.0.7
+
+{{< release-date date="2023-10-26" >}}
+
+For a full list of pull requests and changes in this release, refer to the relevant GitHub milestones:
+
+- [docker/cli, 24.0.7 milestone](https://github.com/docker/cli/issues?q=is%3Aclosed+milestone%3A24.0.7)
+- [moby/moby, 24.0.7 milestone](https://github.com/moby/moby/issues?q=is%3Aclosed+milestone%3A24.0.7)
+
+### Bug fixes and enhancements
+
+* Ensure that overlay2 layer metadata is correct. [moby/moby#46703](https://github.com/moby/moby/pull/46703)
+* Fix "Rootful-in-Rootless" Docker-in-Docker on systemd versions 250 and later. [moby/moby#46626](https://github.com/moby/moby/pull/46626)
+* Fix `docker.socket` not getting disabled when uninstalling the `docker-ce` RPM package. [docker/docker-ce-packaging#852](https://github.com/docker/docker-ce-packaging/pull/852)
+* Fix `dockerd-rootless-setuptools.sh` when username contains a backslash. [moby/moby#46407](https://github.com/moby/moby/pull/46407)
+* Fix a bug that would prevent network sandboxes to be fully deleted when stopping containers with no network attachments and when `dockerd --bridge=none` is used. [moby/moby#46702](https://github.com/moby/moby/pull/46702)
+* Fix a bug where context cancellation could interrupt container restart. [moby/moby#46697](https://github.com/moby/moby/pull/46697)
+* Fix an issue where libnetwork would accept a ChildSubnet with a bigger mask than its parent subnet. [moby/moby#46482](https://github.com/moby/moby/pull/46482)
+* Fix data corruption with zstd output. [moby/moby#46709](https://github.com/moby/moby/pull/46709)
+* Fix the conditions under which the container's MAC address is applied. [moby/moby#46478](https://github.com/moby/moby/pull/46478)
+* Improve the performance of the stats collector. [moby/moby#46448](https://github.com/moby/moby/pull/46448)
+
+### Packaging updates
+
+* Add support for Fedora 39 and Ubuntu 23.10. [docker/docker-ce-packaging#940](https://github.com/docker/docker-ce-packaging/pull/940), [docker/docker-ce-packaging#955](https://github.com/docker/docker-ce-packaging/pull/955)
+* Upgrade BuildKit to [`v0.11.7`](https://github.com/moby/buildkit/releases/v0.11.7). [moby/moby#46441](https://github.com/moby/moby/pull/46441)
+* Upgrade Go to `go1.20.10`. [docker/docker-ce-packaging#951](https://github.com/docker/docker-ce-packaging/pull/951)
+* Upgrade containerd binary to `v1.7.6`. [moby/moby#46103](https://github.com/moby/moby/pull/46103)
+* Upgrade the `containerd.io` package to `v1.6.24`.
```

Comment:
Wasn't sure where to point these links to (and didn't find one for containerd.io)

Reply:
I think it's fine to not link to a commit, but we can link the containerd version to their release-notes; https://github.com/containerd/containerd/releases/tag/v1.6.24